src/migrations/tasks/20231102143211_cwu-opp-under-review.ts

```diff
@@ -0,0 +1,58 @@
+import { makeDomainLogger } from "back-end/lib/logger";
+import { console as consoleAdapter } from "back-end/lib/logger/adapters";
+import Knex from "knex";
+
+const logger = makeDomainLogger(consoleAdapter, "migrations");
+
+enum CWUOpportunityStatus {
+  Draft = "DRAFT",
+  UnderReview = "UNDER_REVIEW",
+  Published = "PUBLISHED",
+  Evaluation = "EVALUATION",
+  Awarded = "AWARDED",
+  Suspended = "SUSPENDED",
+  Canceled = "CANCELED"
+}
+
+enum PreviousCWUOpportunityStatus {
+  Draft = "DRAFT",
+  Published = "PUBLISHED",
+  Evaluation = "EVALUATION",
+  Awarded = "AWARDED",
+  Suspended = "SUSPENDED",
+  Canceled = "CANCELED"
+}
+
+export async function up(connection: Knex): Promise<void> {
+  await connection.schema.raw(
+    ' \
+    ALTER TABLE "cwuOpportunityStatuses" \
+    DROP CONSTRAINT "cwuOpportunityStatuses_status_check" \
+  '
+  );
+
+  await connection.schema.raw(` \
+    ALTER TABLE "cwuOpportunityStatuses" \
+    ADD CONSTRAINT "cwuOpportunityStatuses_status_check" \
+    CHECK ("status" IN ('${Object.values(CWUOpportunityStatus).join("','")}')) \
+  `);
+  logger.info("Completed modifying CWUOpportunityStatuses table.");
+}
+
+export async function down(connection: Knex): Promise<void> {
```

Comment:
Also, I don't know how opinionated you are on this, can we update under review statuses to a valid status when migrating down, probably draft? I'm not able to easily migrate down without deleting records

Reply:
good suggestion: https://github.com/bcgov/digital_marketplace/pull/430/commits/7f3ecefc86c12149ca5e4424dc303f2aceba6987